view/frontend/web/js/applepay/api.js

```diff
@@ -194,16 +194,16 @@ define(
                         return false;
                     }
 
-                    let shippingMethods = [];
+                    var shippingMethods = [];
                     this.shippingMethods = {};
 
                     // Format shipping methods array.
-                    for (let i = 0; i < result.length; i++) {
+                    for (var i = 0; i < result.length; i++) {
                         if (typeof result[i].method_code !== 'string') {
                             continue;
                         }
 
-                        let method = {
+                        var method = {
                             identifier: result[i].method_code,
                             label: result[i].method_title,
                             detail: result[i].carrier_title ? result[i].carrier_title : "",
```

Comment:
This function should follow the single var declaration pattern. `var` declarations are scattered, the scope of `let` is block the scope of `var` is function. In this refactoring, the scope is being changed and there could be issues due to hoisting.